frontend/src/routes/Video.tsx

```diff
@@ -193,11 +195,23 @@ const VideoPage: React.FC<Props> = ({ event, realm, id, basePath }) => {
 type PendingEventPlaceholderProps = {
     event: SyncedEvent;
     startTime: Date;
+    rerenderParent: () => void;
 };
 
-const PendingEventPlaceholder: React.FC<PendingEventPlaceholderProps> = ({ event, startTime }) => {
+const PendingEventPlaceholder: React.FC<PendingEventPlaceholderProps> = ({
+    event,
+    startTime,
+    rerenderParent,
```

Comment:
I would call this somewhat more generic, like `onStart`.

Reply:
good point